docs/pvcscale.md

```diff
@@ -0,0 +1,221 @@
+particular cluster, so it is necessary to adapt `PVCSCALE_MAXPODS` to cluster capacity and
+execute it accordingly. 
+Choosing `PVCSCALE_MAXPODS` to be high can lead that many Pods/PVCs are created what can affect cluster limits and peformance.
+
+### PVCSCALE_POD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+
+Pod container image to use for Pods that are pvscale Pods. This image will be used to
+start Pod and bound PVC to it.
+
+### PVCSCALE_STEPSIZE
+Default: `1`
+
+Number of Pods for cluster loader will create before waiting for Pods to become running.
+PVCScale test creates one Pod/PVC at time.
+
+### PVCSCALE_PAUSE
+Default: `0`
+
+Period of time (in seconds) for cluster loader to pause after creating Pods and 
+waiting for them to be running.PVCScale does not wait on Pods to be `READY` state
+before it creates new ones.
+
+### PVCSCALE_STORAGE_SIZE
+Default: `1Gi`
+
+This is default PVC size. If specified as `1Gi` then all Pods will get `1Gi` PVC
+mounted to `/mnt/pvcmount` inside Pod.
+
+### PVCSCALE_STORAGECLASS
+Default: `none`
+
+It is important to have working storage class prior starting PVCScale test.
+`PVCSCALE_STORAGECLASS` parameter is name of storage class which one plan to
+use, any storage class supporting dynamic storage provisioning will work.
+
+### ACCESS_MODES
+Default: `ReadWriteOnce`
+
+`ACCESS_MODES` is access_mode for PVC inside Pod. This field does not
+affect the test itself and might be removed in future. 
+
+
+### Smoke test variables
+
+```
+PVCSCALE_TEST_PREFIX=pvcscale_smoke
+PVCSCALE_CLEANUP=true
+PVCSCALE_BASENAME=pvcscale
+PVCSCALE_MAXPODS=10
+PVCSCALE_POD_IMAGE="quay.io/openshift-scale/scale-ci-workload"
+PVCSCALE_STEPSIZE=1
+PVCSCALE_PAUSE=0
+```
+
+An longer example of variables can be found
+[here](https://gist.githubusercontent.com/ekuric/6038cb7437d58679de00d31b0365ce4c/raw/d7570f0a4499c6f9d9a395ecdd969bf3bc8b59b3/gistfile1.txt)
+
+### Test results 
+PVCScale test creates first the number of Pods specified in `PVCSCALE_MAXPods`
+variable. Once all Pods are in `READY` state with PVC in `Bound` status,  PVCScale test 
+collects data for all Pods when they are started, and from this information we can then see
+how long it takes to start Pods from Pod1..PodN.
+
+The output will be like one below 
+```
+Pod-Ready
+07:56:29
+07:56:30
+07:56:31
+07:56:32
+07:56:35
+07:56:36
+07:56:39
+07:56:39
+07:56:44
+07:56:46
+``` 
+In this example are visible times when Pods were in `READY` state, where the
+first Pod is started at `07:56:29` and last Pod at `07:56:46`. The time in this output
+is day time when test was executed, and this will be changed in future versions
+of this test to be independent of time when test was ran. In this case the
+difference between last pod start time and first pod start time is total time
+necessary for Pods to be in `READY` state with PVC in `Bound` status.
+
+### Future work 
```

Comment:
What about tracking these as an Issue instead of in the readme?

Reply:
ack, #63 and #64